### ISSUE TITLE
Changes stopped being reported after the first change event

### ISSUE THREAD

**Issue by POPCORNS1068** [paulmillr/chokidar#742]
**Describe the bug**
Chokidar stops reporting a file's changes after the reporting first change event.

**Versions (please complete the following information):**
* Chokidar version 2.0.4
* Node version 10.6.0
* OS: Arch Linux

**To Reproduce**
Steps to reproduce the behavior.
1. Watch a **single file**
2. Modify it (the change would be reported)
3. Modify it again

From here no changes will be reported when the target is updated.

Example:
```
const chokidar = require("chokidar");
const watcher = chokidar.watch("test.txt");

watcher.on("change", filepath => {
    console.log("Update:", filepath);
});
```

**Expected behavior**
The watcher should report changes.

**Additional context**
* Renaming the file, then renaming it back avoids this.
* I would be very surprised if this is the expected behavior.

**Comment by jsantell** [comment]
Seeing this on Debian, probably the same as #237

**Comment by strets123** [comment]
{ useFsEvents: false, atomic: true } 

in the options appears to be a workaround.

**Comment by mitar** [comment]
@jsantell It seems this issue happens no matter if `atomic` is set to `true` or `false`? On Linux, I just tried with `chokidar-cli` and I modified it so that it had `atomic` explicitly set to `true` and another time to `false` and in both cases change was detected only the first time.

**Comment by mitar** [comment]
I think I fixed this in #791.

**Comment by thor** [comment]
Per https://github.com/paulmillr/chokidar/pull/791#issuecomment-496488173 this should be re-opened.
